src/items/traits.md

```diff
@@ -93,7 +93,7 @@ Object safe traits can be the base trait of a [trait object]. A trait is
 # use std::rc::Rc;
 # use std::sync::Arc;
 # use std::pin::Pin;
-// Examples of object safe methods.
+// Examples of trait-object safe methods.
```

Comment:
```suggestion
// Examples of dyn-compatible methods.
```

Though strictly speaking we haven't defined "dyn compatible" for methods, only for traits ("type mismatch"). We could be more explicit.